qfieldsync/gui/layers_config_widget.py

```diff
@@ -256,3 +259,12 @@ def apply(self):
 
         if is_project_dirty:
             self.project.setDirty(True)
+
+    def _on_message_bus_messaged(self, msg: str) -> None:
+        if msg != "layer_config_saved":
```

Comment:
Would it make sense to document where this is sent from/to?
Mostly because it's just a magic string to document somewhere.